src/browser-runner/platform-qbrt/shell/main.js

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2017 Mozilla
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License. */
+
+/* global Components */
+
+const { classes: Cc, interfaces: Ci, utils: Cu } = Components;
+const { Runtime } = Cu.import('resource://qbrt/modules/Runtime.jsm', {});
+const { Services } = Cu.import('resource://gre/modules/Services.jsm', {});
+
+
+// TODO: Pass correct values as command line arguments to qbrt
+const searchParams = [
+  'runnerConnId=%2Fv1%2Frunner%23k4cFvGGdjECa9trEAAAB',
+  'winId=2d12c846-3712-411b-8efb-f810683aa3ac',
+  'os=darwin',
+  'platform=qbrt',
+  'apiVersion=v1',
+  'hostname=localhost',
+  'port=9000',
+].join('&');
```

Comment:
Let's get these fixed before landing. The hardcoded runnerConnId will most likely never be the same one.